astro-docs/src/content/docs/technologies/build-tools/docker/introduction.mdoc

```diff
@@ -0,0 +1,64 @@
+---
+title: Overview of the Nx Docker Plugin
+description: The Nx Plugin for Docker contains executors and utilities for building and publishing docker images within an Nx workspace.
+keywords: [docker]
```

Comment:
```suggestion
keywords: [docker]
sidebar:
  label: Introduction
```

this will put 'introduction' as the sidebar title otherwise astro will take the `title` property